src/razor/src/semantic/semanticTokensRangeHandler.ts

```diff
@@ -18,8 +17,8 @@ export class SemanticTokensRangeHandler {
         any
     > = new RequestType(SemanticTokensRangeHandler.getSemanticTokensRangeEndpoint);
     private emptySemanticTokensResponse: ProvideSemanticTokensResponse = new ProvideSemanticTokensResponse(
-        new SemanticTokensResponse(new Array<number>(), ''),
-        null
+        new Array<Array<number>>(),
+        -5
```

Comment:
@davidwengier adding a number like -5 combined with the following commit in dotnet/razor would fix the new bug introduced for VS code that is having all razor components show up as red

https://github.com/dotnet/razor/commit/889f03b9bcf7c05cfc7e9fead16387a6662eff8f

Reply:
We have access to the right version number, and that would be preferable to a magic number like -5. I'm on my phone so can't get a link, but there is a document synchronizer here too.

The other option is to add a language server feature option on the server side, and stop us bothering to request semantic tokens on vscode at all, and then just delete this whole folder.

Reply:
Addressed feedback in https://github.com/dotnet/vscode-csharp/pull/6304/commits/f1d84b203a87fdb7f49c0927e5e98e578ab6bc31